service/src/main/resources/application.yml

```diff
@@ -38,7 +38,7 @@ spring:
           maxAge: 0
           mustRevalidate: true
         useLastModified: false
-      staticLocations: classpath:/api/
+      staticLocations: classpath:/api/,classpath:/schema/development/
```

Comment:
In the future, this value could be configured using the application property that specifies the active schema for a deployment. (which is currently stored in `catalog.schema.basePath`).